doc/integrator/filter.rst

```diff
@@ -3,13 +3,53 @@
 Filter (Querier)
 ================
 
+Filterable layers
+-----------------
+Per default, layers are not filterable.
+If you wish to provide the filter functionality, proceed as follows:
+
+* ensure that the functionality ``filterable_layers`` is made available in your project
+  template files:
+
+  - if you have made the default functionalities of ``CONST_vars.yaml`` available in your
+    project templates, then ``filterable_layers`` is already available in your project templates.
+    Check your project vars file, if the ``update_paths`` contains the entry
+    ``functionalities.available_in_templates``, then you are using the default definition of
+    functionalities as provided by ``CONST_vars.yaml``
+
+  - alternatively, you can define a custom ``functionalities.available_in_templates`` in your
+    project vars file. If you do this, this needs to contain the entry ``filterable_layers``.
+
+* if you wish to provide layers for filtering independently of user authorization (in particular,
+  for not-logged-in users), add the layer names as follows to your project vars file:
+
+.. code:: yaml
+
+    functionalities:
+        anonymous:
+            filterable_layers:
+            - my_layer_name_1
+            - my_layer_name_2
+
+* if you wish to provide layers for filtering only for specific user roles, define this in the
+  admin interface as follows:
+
+  - add a functionality ``filterable_layers``; as value provide the layer name.
+
+  - edit the role that shall be able to use this filter, and activate the corresponding
+    functionality checkbox.
+
+* note that if there are functionalities associated to a role, then users with this role will
+  not have access to any of to the "anonymous" functionalities. Instead, you will need to activate
+  in the admin interface all functionalities that this role shall be able to use.
```

Comment:
in the admin interface or in the functionalities.registered variable

Reply:
rewritten